site/cypress/integration/autograding_status_1.spec.js

```diff
@@ -15,7 +15,7 @@ skipOn(Cypress.env('run_area') === 'CI', () => {
         });
 
         afterEach(() => {
-            cy.logout();
+            cy.logout(true);
         });
```

Comment:
Do we even need to logout? Per the cypress docs, it looks like it automatically clears localStorage, cookies, and sessions before each test, which would automatically force a logout. We are left with a bunch of "dangling" sessions between each test, but whatever, that's not really something to worry about.

Reply:
Ideally there would be no need to logout, but sometimes it doesn't actually clear everything between test cases (which leads to an error when it tries to login when it's already logged in) - I'm not exactly sure why it happens but I know that it happens sometimes when an XHR request spills over from the previous test case - https://github.com/cypress-io/cypress/issues/216 + https://github.com/cypress-io/cypress/issues/686.

Reply:
Hm, alright. Can you move this `afterEach` to live in `site/cypress/support/index.js` so that it's just defined once (though I don't remember if that'll operate after each test, or after each outer describe...)?

Reply:
The `api/auth.spec.js` tests don't log in or out of the site using the standard method; I wasn't able to find a way to have a global afterEach that excludes certain test files.

Reply:
Given that they don't really ever make use of anything out of cypress, I'd be in-favor of moving them out of cypress then if it would simplify other things.

We can move them into `site/tests/api` and then use regular `jest` and such on running them.